src/types/BannerTypes.ts

```diff
@@ -20,9 +20,9 @@ export type BannerTargeting = {
 };
 
 export type BannerTestTracking = {
-    abTestName: string;
-    abTestVariant: string;
-    campaignCode: string;
+    abTestName?: string;
```

Comment:
The `addTrackingParams` function expects these to be set.
I know not everything is AB tested but we've always set test/variant names even where there's only a control - it's nice to have this in the analytics

Reply:
Makes sense, I'll add some default values for puzzles.

Reply:
👍 thanks!